README.md

```diff
@@ -1,8 +1,125 @@
 SolidusImporter
 ===============
 
```

Comment:
In my opinion, a small introduction is nice and useful. I would keep at least a description line.